config/samples/app_retrievalqachain_knowledgebase.yaml

```diff
@@ -78,7 +78,7 @@ spec:
     Use the following pieces of context to answer the question at the end. If you don't know the answer, just say that you don't know, don't try to make up an answer.  
     
     {{.context}}
-    
+    --------
```

Comment:
it'll work better if add this separater?

Reply:
In some cases, there are improvements...